sound/soc/sof/intel/Kconfig

```diff
@@ -300,7 +300,6 @@ if SND_SOC_SOF_HDA_COMMON
 
 config SND_SOC_SOF_HDA_LINK
 	bool "SOF support for HDA Links(HDA/HDMI)"
-	depends on SND_SOC_SOF_NOCODEC_SUPPORT=n
```

Comment:
@plbossart does this mean we can also add the HDMI PCMs in the LNL nocodec topology?

Reply:
no. What this means is that we can add SSP/DMIC with HDaudio support in the nocodec build.

HDMI is still dependent on hdac_hda.c and that's controlled by the HDA_AUDIO_CODEC part.